docs/fundamentals/runtime-libraries/snippets/System/Span/Overview/csharp/program.cs

```diff
@@ -85,8 +88,8 @@ public class ProgramB
     public static void WorkWithSpans()
     {
         // Create a span over an array.
-        var array = new byte[100];
-        var arraySpan = new Span<byte>(array);
+        byte[] array = new byte[100];
+        Span<byte> arraySpan = new(array);
```

Comment:
Same comment as above:

```suggestion
        Span<byte> arraySpan = stackalloc new byte[100];
```